### PR TITLE
[service.kn.switchtimer] 2.0.1

### PR DESCRIPTION
**Pull request by b-jesch**
Update information:

2.0.1
- Profile paths for timer.json considered

2.0.0
- complete Redesign of source code
- Timers are stored into a JSON structured file independent from skin
- IMPORTANT: Timers use now skin properties instead of skin labels as suggested by phil65:  https://github.com/xbmc/repo-scripts/pull/252#r77217118

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [ ] Each add-on submission should be a single commit with using the following style: [sciprt.foo.bar] v1.0.0

### CODE REVIEW

**Comment by enen92** [comment]
Can you rename the pull request please to match your commits?

File: service.kn.switchtimer/addon.xml [hunk 1]
```diff
@@ -1,23 +1,23 @@
 <?xml version='1.0' encoding='UTF-8' standalone='yes'?>
-<addon id="service.kn.switchtimer" name="KN Switchtimer Service" provider-name="Birger Jesch" version="1.2.4">
+<addon id="service.kn.switchtimer" name="KN Switchtimer Service" provider-name="Birger Jesch" version="2.0.1">
     <requires>
-        <import addon="xbmc.addon" version="16.0.0"/>
-        <import addon="xbmc.python" version="2.24.0"/>
-        <import addon="xbmc.json" version="6.32.4"/>
+        <import addon="xbmc.addon" version="14.0.0"/>
+        <import addon="xbmc.python" version="2.20.0"/>
```
Comment (enen92) [inline service.kn.switchtimer/addon.xml]: 2.24.0 was correct for `xbmc.python` since this is in Jarvis branch. Why did you change it for the Isengard value?

File: service.kn.switchtimer/addon.xml [hunk 2]
```diff
@@ -1,23 +1,23 @@
 <?xml version='1.0' encoding='UTF-8' standalone='yes'?>
-<addon id="service.kn.switchtimer" name="KN Switchtimer Service" provider-name="Birger Jesch" version="1.2.4">
+<addon id="service.kn.switchtimer" name="KN Switchtimer Service" provider-name="Birger Jesch" version="2.0.1">
     <requires>
-        <import addon="xbmc.addon" version="16.0.0"/>
-        <import addon="xbmc.python" version="2.24.0"/>
-        <import addon="xbmc.json" version="6.32.4"/>
+        <import addon="xbmc.addon" version="14.0.0"/>
```
Comment (enen92) [inline service.kn.switchtimer/addon.xml]: Please drop this one it is not needed

**Comment by b-jesch** [comment]
All done.

**Comment by enen92** [comment]
Thanks

**Comment by b-jesch** [comment]
Thank You for the quick processing.
